### PR TITLE
Locator changes based upon ODF version

### LINKED ISSUE #6566: Change locator based upon ODF version and not OCP version
As we know that the ODF console is now a separate repository, so the locators couldn't be used baseb upon OCP version, rather we need to use ODF version. In this case, the OCP is 4.11, and the ODF is 4.10. So according to our code, it's checking the locators under OCP 4.11, however, it has different locators coming from the ODF console.

For test- `tests/ui/test_validation_ui.py`

1st- For overview under validation_4_11
Used locator- //span[normalize-space()='Overview']
New locator- `//a[@data-test-id='horizontal-link-Overview']`

2nd- For backingstorage_breadcrumb under validation_4_11
Used locator- //a[normalize-space()='BackingStores']
New locator- `//a[normalize-space()='[noobaa.io](http://noobaa.io/)~v1alpha1~BackingStore']`

`ocs-ci results for OCS4-10-Downstream-OCP4-11-AZURE-IPI-3AZ-RHCOS-3M-3W-tier1-or-tier_after_upgrade-post-upgrade (BUILD ID: 4.10.7-1 RUN ID: 1665163364)`

https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-deploy-ocs-cluster-prod/5637/consoleFull

### PR DESCRIPTION
**Pull request by am-agrawa**
Signed-off-by: am-agrawa <amagrawa@redhat.com>

fixes #6566

### CODE REVIEW

**Review by keesturam — APPROVED** [review]
LGTM

File: ocs_ci/ocs/ui/validation_ui.py [hunk 1]
```diff
@@ -433,16 +451,53 @@ def odf_storagesystems_ui(self):
...
+                )
+            else:
+                self.do_click(
+                    self.validation_loc["blockandfile"], enable_screenshot=True
+                )
         if not config.DEPLOYMENT.get("external_mode"):
             if not config.ENV_DATA["mcg_only_deployment"]:
                 logger.info("Click on 'BlockPools' tab")
-                self.do_click(self.validation_loc["blockpools"], enable_screenshot=True)
+                if (
```
Comment (keesturam) [inline ocs_ci/ocs/ui/validation_ui.py:489]: What happens with 4.12?

Comment (am-agrawa) [reply ocs_ci/ocs/ui/validation_ui.py:489]: For 4.12 or any other version, it will execute the line in the else part.

**Review by sidhant-agrawal — APPROVED** [review]

**Review by keesturam — APPROVED** [review]

**Review by Avilir — APPROVED** [review]

**Comment by am-agrawa** [comment]
@vkathole Looks like something is messedup with your commit. Could you pls check?

**Pull request merged**